app/schemas.py

```diff
@@ -272,6 +272,8 @@ def handle_template_merge(self, in_data):
             if in_data['template']['template_type'] == 'email':
                 in_data['subject'] = template.replaced_subject
             in_data.pop('personalisation', None)
+        else:
+            in_data['body'] = in_data['template']['content']
```

Comment:
Oh and I should have mentioned why any of this is necessary at all. Marshmallow doesn't seem to play nice with python properties, hence the need for pre and post jiggery pokery.